hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestRatisPipelineCreateAndDestroy.java

```diff
@@ -103,7 +108,9 @@ public void testPipelineCreationOnNodeRestart() throws Exception {
     } catch (IOException ioe) {
       // As now all datanodes are shutdown, they move to stale state, there
       // will be no sufficient datanodes to create the pipeline.
-      Assert.assertTrue(ioe instanceof InsufficientDatanodesException);
+      Assert.assertTrue(ioe instanceof SCMException);
+      Assert.assertTrue(((SCMException) ioe).getResult()
```

Comment:
Can we use assertEquals with some message so that when it fails, we can see the difference in the ResultCode.

Reply:
Sure